roles/logs_client/README.md

```diff
@@ -13,15 +16,28 @@
     ansible-playbook -u [admin] -l jumphost single_group_playbooks/logs.yml
 ```
 
-## I. Prerequisites
+## I. Definitions
+
+List of interesting variables
+
+ - `logs_class_list` - a list of all available classes (like 'development', 'research', 'diagnostic' ... )
```

Comment:
Looking at the `Compile the list of logs server names` task the `logs_class_list` appears to be a list of IP addresses for all logs servers of a certain logs class. That is something else than `a list of all available classes (like 'development', 'research', 'diagnostic' ... )`; I'm a little confused...